ext/node/polyfills/net.ts

```diff
@@ -1598,7 +1598,7 @@ Socket.prototype.destroySoon = function () {
 
 Socket.prototype._unrefTimer = function () {
   // deno-lint-ignore no-this-alias
-  for (let s = this; s !== null; s = s._parent) {
+  for (let s = this; s != null; s = s._parent) {
```

Comment:
Are these unrelated changes?

Reply:
Yes, `_parent` can be undefined (not just null) when s is `JSStreamSocket`